lib/linguist/languages.yml

```diff
@@ -4056,6 +4078,7 @@ Scheme:
   color: "#1e4aec"
   extensions:
   - ".scm"
+  - ".sch"
```

Comment:
Travis shat bricks the moment I submitted the PR... any idea what's up with that? 😕